src/host/selectionInput.cpp

```diff
@@ -184,7 +184,7 @@ COORD Selection::WordByWordSelection(const bool fReverse,
 
     // track whether we failed to move during an operation
     // if we failed to move, we hit the end of the buffer and should just highlight to there and be done.
-    bool fMoveSucceeded = false;
+    bool fMoveSucceeded;
 
     // determine if we're highlighting more text or unhighlighting already selected text.
     bool fUnhighlighting;

```

Comment:
please do not introduce unrelated changes. If I recall correctly, we've had this discussion before?

Reply:
It's hard enough to review a pull request that changes 174 files when the _only_ changes are the ones you tell us about. It's impossible to review them when there could be other random changes inside it. :smile: